core/src/main/java/org/mule/runtime/core/api/store/DeserializationPostInitialisable.java

```diff
@@ -4,7 +4,7 @@
  * license, a copy of which has been included with this distribution in the
  * LICENSE.txt file.
  */
-package org.mule.runtime.core.util.store;
+package org.mule.runtime.core.api.store;
```

Comment:
This is the first iteration of API clean up. This is required to be API on the current state of the code. After the initial clean up, the remaining exported packages will be reviewed again and cleaned if possible